src/WebHook/UpdateLineItemDto.php

```diff
@@ -31,7 +31,7 @@ class UpdateLineItemDto implements JsonSerializable
     public const STATUS_ERROR = 'error';
     public const STATUS_IGNORED = 'ignored';
 
-    public const NAME = 'RemoteDeliveryPublicationFinished';
+    public const NAME = 'oat\\taoPublishing\\model\\publishing\\event\\RemoteDeliveryCreatedEvent';
```

Comment:
I assume this is a class namespace on TAO side. I really don't think that we should rely on it, it's very fragile. What do you think?

Reply:
Agree, same question from me, what is the gain to use FQCN, instead of just Classname?

Reply:
Nothing more to say, I'm with the guys :)

Reply:
Agree. Let's include @bartlomiejmarszal on this discussion and see what we can do.

Reply:
In my opinion we should use FQN in configuration. We should never hardcore external webhook in our code

Reply:
Sorry, but the problem is not a configuration, we would stiil need to have a config in our context relying on TAO context this doesn't solve the problem.

Reply:
The simple-roster is relying on external application that can call your api. 
So we have 2 choices IMO:
1. simple-roster adapts to Tao webhook
2. Tao webhook adapts to simple-roster
Either of choices should include configuration to allow adapt to each other without changes in code like external namespaces. 
This is why my proposal is:
- we should create in your code a service that will accept `webhoook configuration`, that will allow Tao web hooks work in simple roster. 
We encounter already this problem with payload of this request.

Reply:
No Simple Roster is not Relying on any kind of appiication. TAO is relying on the Simple Roster via WebHook.
About the 2 options what we would like you to consider here is to remove the FQDNs. Reason: Everybody that will consume this webhooks will have to know about the TAO context in their domains. Just imagine if you have to do it for external clients. they will know about the TAO event classes. is that what you want?

Again the configuration it won't sovle the problem because somewhere into our application (it doesn`t matter if is code or env var or .yaml) will know the fully qualified domain name of your event. So this definitely doesn't solve the issue.

For me what you have to think is about your webhook interface. Do you really want to expose your FQDNs on your webhook interface? because this is your domain. You should decide what would be the best? We're just a consumer.

Reply:
> Everybody that will consume this webhooks will have to know about the TAO context in their domains. Just imagine if you have to do it for external clients

Yes. This is exactly what I'm expecting. Anybody who consume webhook call should exactly know what he can expect from this call, which event he is subscribing to and what is this call payload. This is why I find extremely important to keep this adaptation in configuration or library.

Reply:
There is no gain to have it as a configuration. One the implementation is done, It will not change, This is not a configuration. This is a constant.

Reply:
i would provide my the following from my perspective : 
 - move the name to a .env but leave a default 
 - let's not use FQN because you add coupling to the library and we don't want to do that,
 - i would document this webhook / name or even interface expected inside the readme
 because this is not in a authoring ownership,
 the simple roaster team should decide the way of implementing this according to the company standards

Reply:
> Yes. This is exactly what I'm expecting. Anybody who consume webhook call should exactly know what he can expect from this call, which event he is subscribing to and what is this call payload. This is why I find extremely important to keep this adaptation in configuration or library.
Do not get me wrong but could you explain to me what this has to do with being configurable or not? I do not see your point here.

Regarding the library, I believe that it makes sense, but It is out of scope now. Do you plan to create a new library to encapsulate this as TAO is the webHook provider? I do not see this reasonable.

Reply:
If you are going to introduce hardcoded expectations such as event name you are internally coupling with external application behaviour. If you doing this in configuration you are separating your internal business logic from any external coupling.

Reply:
Guys, after team consultation regarding this event we agreed that it's too risky to change name of this event to simple class name. We have to ensure we comply with backwards compatibility so existing consumers of this webhook will not loose functionality.